File: Algotoolbox/week4/linearsearch.py
from typing import List


def linearSearch(target, array:List):
    n = len(array)
    for i in array:
        if array[i] == target:
            return i
    else:
        return -1


def main():
    arr = [1,5,8,9,6,4,7,5,2,3,5]
    print(linearSearch(9,arr))

if __name__ == '__main__':
    main()       

File: Algotoolbox/week4/linearsearch.cpp
#include <bits/stdc++.h>
using namespace std;
int linearSearch(int target,vector<int>&arr ){
    int n = arr.size();
    for(int i=0; i < n ; i++){
        if(arr[i] == target){
            return i-1;
        }
    }
    return -1;
}
int main(){
    vector<int> arr = {25,4,5,9,7,15};
    int ans = 4 < 5;
    cout << ans;
    return 0;
}